Corona-Warn-App/src/main/res/layout/validation_start_fragment.xml

```diff
@@ -0,0 +1,307 @@
+                app:layout_constraintStart_toStartOf="@+id/body"
+                app:layout_constraintTop_toBottomOf="@+id/country_layout">
+
+                <com.google.android.material.textfield.TextInputEditText
+                    android:id="@+id/date_picker"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:clickable="true"
+                    android:focusable="false"
+                    android:inputType="none" />
+
+            </com.google.android.material.textfield.TextInputLayout>
+
+            <TextView
+                android:id="@+id/date_info"
+                style="@style/TextAppearance.MaterialComponents.Caption"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="6dp"
+                android:layout_marginEnd="@dimen/spacing_normal"
+                android:text="@string/validation_start_date_info"
+                app:layout_constraintEnd_toEndOf="@id/body"
+                app:layout_constraintStart_toStartOf="@id/body"
+                app:layout_constraintTop_toBottomOf="@id/date_layout" />
+
+            <ImageView
+                android:id="@+id/date_info_icon"
+                android:layout_width="20dp"
+                android:layout_height="20dp"
+                android:layout_marginStart="10dp"
+                android:layout_marginEnd="14dp"
+                android:importantForAccessibility="no"
+                app:layout_constraintEnd_toEndOf="@id/body"
+                app:layout_constraintStart_toEndOf="@id/date_info"
+                app:layout_constraintTop_toTopOf="@id/date_info"
+                app:srcCompat="@drawable/ic_info"
+                app:tint="@color/colorPrimary" />
+
+            <TextView
+                android:id="@+id/certificate_proof"
+                style="@style/body1"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="26dp"
+                android:text="@string/validation_start_certificate_proof"
+                app:layout_constraintEnd_toEndOf="@id/body"
+                app:layout_constraintStart_toStartOf="@id/body"
+                app:layout_constraintTop_toBottomOf="@id/date_info" />
+
+            <TextView
+                android:id="@+id/note_subtitle"
+                style="@style/TextAppearance.MaterialComponents.Subtitle2"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="24dp"
+                android:text="@string/validation_start_note_subtitle"
+                app:layout_constraintEnd_toEndOf="@id/body"
+                app:layout_constraintStart_toStartOf="@id/body"
+                app:layout_constraintTop_toBottomOf="@id/certificate_proof" />
+
+            <include
```

Comment:
bullets alignment should be shifted a little bit , currently it is aligned to the title